internal/provider/provider.go

```diff
@@ -21,25 +23,27 @@ type Provider struct {
 }
 
 // ProviderModel describes the provider data model.
-type ProviderModel struct {
+type ProviderModel struct { //nolint:revive
 	Repo      types.String `tfsdk:"repo"`
 	BasicAuth types.String `tfsdk:"basic_auth"`
 
 	// TODO: default base image
+	// TODO: default base image per importpath
```

Comment:
I'm curious what you have in mind here, since this seems like a case where folks would naturally specify it in `ko_build` 🤔

Reply:
Probably resolve?

Reply:
Yep, you're right. Removing this.